src/components/AppSidebar/AppSidebar.vue

```diff
@@ -123,36 +154,55 @@
 				</div>
 
 				<!-- sidebar details -->
-				<div v-if="!empty" :class="{ 'app-sidebar-header__desc--with-star': canStar, 'app-sidebar-header__desc--with-subtitle': subtitle && !titleEditable, 'app-sidebar-header__desc--editable': titleEditable && !subtitle, 'app-sidebar-header__desc--with-subtitle--editable': titleEditable && subtitle}" class="app-sidebar-header__desc">
+				<div v-if="!empty"
+					:class="{
+						'app-sidebar-header__desc--with-star': canStar,
+						'app-sidebar-header__desc--with-subtitle': subtitle && !titleEditable,
+						'app-sidebar-header__desc--editable': titleEditable && !subtitle,
+						'app-sidebar-header__desc--with-subtitle--editable': titleEditable && subtitle
+					}"
+					class="app-sidebar-header__desc">
 					<!-- favourite icon -->
-					<a v-if="canStar"
-						:class="{ 'icon-starred': isStarred&& !starLoading, 'icon-star': !isStarred && !starLoading, 'icon-loading-small': starLoading }"
-						class="app-sidebar-header__star"
-						@click.prevent="toggleStarred" />
+					<div v-if="canStar" class="app-sidebar-header__star-action">
+						<a :class="{
+								'icon-starred': isStarred && !starLoading,
+								'icon-star': !isStarred && !starLoading,
+								'icon-loading-small': starLoading
+							}"
+							class="app-sidebar-header__star"
+							@click.prevent="toggleStarred" />
+					</div>
 
 					<!-- main title -->
-					<h2 v-if="!titleEditable" v-linkify="title" class="app-sidebar-header__title" />
-					<template v-if="titleEditable">
-						<form
-							class="rename-form"
-							@submit.prevent="onSubmitTitle">
-							<input
-								v-focus
-								class="app-sidebar-header__title-input"
-								type="text"
-								:placeholder="titlePlaceholder"
-								:value="title"
-								@keydown.esc="onDismissEditing"
-								@input="onTitleInput">
-							<button
-								class="icon-confirm"
-								type="submit" />
-						</form>
-					</template>
-					<!-- secondary title -->
-					<p v-if="subtitle.trim() !== ''" class="app-sidebar-header__subtitle">
-						{{ subtitle }}
-					</p>
+					<div class="app-sidebar-header__title">
+						<h2 v-show="!titleEditable"
+							v-linkify="title"
+							class="app-sidebar-header__maintitle"
+							@click.self="editTitle($event)" />
+						<template v-if="titleEditable">
+							<form
+								v-click-outside="() => onSubmitTitle()"
```

Comment:
@raimund-schluessler I just noticed this, we do not forward any event here. :see_no_evil: 
Any reason you did not just do that?

```suggestion
								v-click-outside="onSubmitTitle"
```

Reply:
No, I probably just overlooked it.

Reply:
All good, thanks :)